sdk/appconfiguration/app-configuration/tsconfig.json

```diff
@@ -1,21 +1,12 @@
 {
+  "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "target": "es6",
-    "module": "es6",
-    "moduleResolution": "node",
-    "strict": true,
-    "sourceMap": true,
-    "inlineSources": true,
-    "declarationMap": true,
-    "declaration": true,
+    "target": "es5",
     "declarationDir": "./types",
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "forceConsistentCasingInFileNames": true,
     "outDir": "./dist-esm",
-    "importHelpers": true,
-    "newLine": "LF"
+    "downlevelIteration": true
   },
-  "exclude": ["node_modules", "types/**", "./samples/**/*.ts"],
+  "compileOnSave": true,
```

Comment:
do we want this? I thought we always build through npm/rush?

Reply:
This file comes from what I was able to see here: https://github.com/Azure/azure-sdk-for-js/pull/6285/files I'm ok if we want to remove this setting everywhere! That's an easy change. Let me know if that's what we want.

Reply:
@bterlson thoughts? I am thinking we remove this everywhere.

Reply:
I'll follow your advice and change it. I can change it back if Brian says so too :)

Reply:
I'll push the change. I did it with `find . -type f -exec sed -i '/compileOnSave": true/d' {} \;` at the `sdk` folder. Checked that it was correct with `git diff | grep "^-  \"[^c]"`.

Reply:
Agree on removing, I don't think it serves a purpose. I also disagree with that preference being specified in tsconfig to begin with, but that's a separate story 😁

Reply:
Thank you Brian!